test/utils.ts

```diff
@@ -51,4 +51,4 @@ export function resolvePathFromTestRoot(...args: string[]): string {
 }
 
 // eslint-disable-next-line -- do not rewrite exports
-export { chai, path, fs }
+export {chai, path, fs};
```

Comment:
Ah nice catch on this one whoops

Reply:
ah I just ran `npm run format` :| I need to fix the precommit hooks to ensure that.